library/ssl_msg.c

```diff
@@ -5595,11 +5595,17 @@ static int ssl_tls13_handle_hs_message_post_handshake(mbedtls_ssl_context *ssl)
         if (ssl_tls13_is_new_session_ticket(ssl)) {
 #if defined(MBEDTLS_SSL_SESSION_TICKETS)
             MBEDTLS_SSL_DEBUG_MSG(3, ("NewSessionTicket received"));
-            ssl->keep_current_message = 1;
+            if (ssl->conf->new_session_tickets_enabled ==
+                MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_ENABLED) {
```

Comment:
Not a blocker, but `xxx_enabled` looks like a boolean setting to me, so I'd expect
```suggestion
            if (ssl->conf->new_session_tickets_enabled) {
```

Reply:
`new_session_tickets_enabled` has been removed thus things are different here now.